src/@carbon-info/layout/MainLayout/components/Header/Header.tsx

```diff
@@ -58,7 +58,7 @@ const Header: React.FC = () => {
         {!isMobile && (
           <>
             <Box onClick={() => window.open(`${Path.Footer.Buy}`, "_blank")} className={classes.externalLink}>
```

Comment:
You can also use the CTAButton for the header and style it accordingly (font size, icon size, colour, ...). 

E.g.           
```suggestion
                      <CTAButton
            text="Buy $SWTH"
            link={Path.Footer.Buy}
            textClassName={classes.ctaButtonText}
            iconClassName={classes.ctaButtonIcon}
            buttonClassName={classes.ctaButtonBox}
          />
```